deployment/secure_api_server.py

```
@@ -516,239 +604,241 @@
-            }
         }
-        
+
         response_time = time.time() - start_time
         update_metrics(response_time, success=True)
-        
+
         return jsonify(response)
-        
+
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)